commands/birthday.ts

```diff
@@ -0,0 +1,56 @@
+import { CommandInteraction, Message, User } from 'discord.js'
+import { SlashCommandBuilder } from '@discordjs/builders'
+import moment from 'moment-timezone'
+import prisma from '../utils/prisma'
+
+async function setBday(userId, date) {
+  if (userId === null || date === null) return
+  const user = await prisma.user.upsert({
```

Comment:
Only allow user to create birthday once with create and do a try catch to catch if bday already exists and return them a ephemeral message in the catch